i18n_subsites/i18n_subsites.py

```diff
@@ -104,6 +104,8 @@ def prepare_site_db_and_overrides():
             overrides['THEME_STATIC_DIR'] = posixpath.join(
                 relpath, _MAIN_SETTINGS['THEME_STATIC_DIR'])
             overrides['THEME_STATIC_PATHS'] = []
+        if 'FEED_DOMAIN' not in overrides:
+            overrides['FEED_DOMAIN'] = posixpath.join(main_siteurl, lang)
```

Comment:
This should probably use `settings['FEED_DOMAIN']` instead of `main_siteurl`

Reply:
I changed what you said @avaris, but I think it's not the best idea. If you have `FEED_DOMAIN` set to `SITEURL + '/feeds'`, for example, the link to the feeds for Czech will be like `http://example.com/test/feeds/cz/all.atom.xml`, and this plugin doesn't organize the output that way, so the links are broken.

In this case, when `FEED_DOMAIN` is `None`, it works just fine, but if it's provided it makes things more difficult and unexpected with that change.